### DISCUSSION TITLE
The QR of Blynk not work

### DISCUSSION THREAD

**Discussion by compnew2006** [taligentx/dscKeybusInterface#239] [dscKeybusInterface Discussion]
The QR of Blynk not work 
here no read the QR
![Screenshot_20210424-183126_Blynk](https://user-images.githubusercontent.com/51255434/115975543-4b463080-a52b-11eb-80d5-e7c83b3252a4.jpg)

and here when i try to import the code from photo it's say Not a Valid Blynk QR-code in the bottom
![Screenshot_20210424-183140_Blynk](https://user-images.githubusercontent.com/51255434/115975546-513c1180-a52b-11eb-8a1d-41273416ca52.jpg)

**Comment by kricon** [comment]
Several Android devices/version doesn't want to accept QR codes, discussed in #190 
Try on another device (preferably iOS one), add the project example to the blynk server.

**Comment by compnew2006** [comment]
Ok i will try
